lib/awful/completion.lua

```diff
@@ -154,10 +161,25 @@ function completion.shell(command, cur_pos, ncomp, shell)
         while true do
```

Comment:
Maybe we should also get rid of the `sort -u`, since the shell should specify the order already?